agenta-backend/agenta_backend/services/json_importer_helper.py

```diff
@@ -0,0 +1,22 @@
+import os
+import json
+from typing import Dict
+from pathlib import Path
+
+
+def get_single_prompt_testsets() -> Dict[str, str]:
```

Comment:
rename this method to `get_json` or `import_json` and add a path argument.
Pass the path argument from 

https://github.com/Agenta-AI/agenta/pull/623/files#diff-44602232f1b16f4cacf23dcfdd269cb9c0325418594a4a07a1b6197db07ccbf3R137